tradable-app/src/store/ContextProvider.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+import Context from "./Context";
+
+const ContextProvider = (props) => {
+   const [darkMode, setDarkMode] = useState(true);
+   const [candlesData, setCandlesData] = useState({});
+
+   const setCandles = (data) => {
+      const candlesObj = candlesData;
+      data.map((c) => {
+         if (candlesObj[c.symbol]) {
+            candlesObj[c.symbol].push(c);
+         } else {
+            candlesObj[c.symbol] = [{ ...c }];
+         }
+      });
+      const obj = { ...candlesObj };
+      setCandlesData(obj);
+      // console.log(obj);
+   };
+
+   const getMode = () => {
+      return darkMode ? "darkMode" : "lightMode";

```

Comment:
if its only 2 so true and false can do the work
if its not dark - will be white